crates/core/src/host/wasmer/wasm_instance_env.rs

```diff
@@ -154,13 +154,16 @@ impl WasmInstanceEnv {
     }
 
     /// Provide this environment with information about the current reducer being executed.
-    pub fn set_reducer_id(&mut self, id: u64) {
-        self.reducer_id = id;
+    pub fn set_reducer_name(&mut self, name: String) {
```

Comment:
This is going to collide with Noa's refactor in https://github.com/clockworklabs/SpacetimeDB/pull/417 which moves the setting of this into the place where reducers are called. It's a change I approve of, so I'd like to see this updated to do that. Noa's PR is approved with one tiny change request so hopefully soon.